backend/app/services/groups/create.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Groups
+  class Create < Core::Service
+    def call(params:, user:)
+      @params = params
+      @user = user
+      group.user_id = user.id
+      group.save ? Success(group) : Failure(group.errors)
```

Comment:
Let's move it (and line above) to the separate `create_group` method and use `yield create_group` there